OxfordBodleian/Juel-Jensen/BDLaethd20.xml

```diff
@@ -3520,7 +3521,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </list>
                                 </collation>
                                 
-                                <condition key="good">Slight traces of use: stained by dirt in the lower margin.</condition>
+                                <condition key="good"> 
+                                    Slight traces of use: worn at the spine; 
+                                    stained by dirt in the lower margin until <locus target="#105r"/>. Occasionally smudged ink, 
+                                e.g. on <locus target="#71v"/>. <locus from="109" to="117"/> are stained by a fluid.
+                                    The first and last folio have been stained by the cover. <!--is there a better way to say this?--> The 
```

Comment:
Maybe: The first and last folio are discoloured / show discolourations from the cover?

Reply:
Thank you, I've taken Jacopo's description. No opinion on the crumpling, I followed Massimo's example on that...At some point, apart from entire manuscript revisions, we will need to read the single categories side by side, and edit a bit for a common style, both of description and of what is described. This will probably be easier than listing all the parameters up in the meeting?